src/utils/testSupport/TestingUtils.ts

```diff
@@ -185,7 +193,10 @@ export async function testTurn(
     } else {
         const expectedPrompt: string =
             typeof expectedResponse === 'string' ? expectedResponse : expectedResponse.prompt;
-        if (!expectedPrompt.toLowerCase().startsWith('a:')) {
+        if (
+            !expectedPrompt.toLowerCase().startsWith('a:') &&
+            input.handlerInput.requestEnvelope.request.type !== 'CanFulfillIntentRequest'
```

Comment:
Why is this check required on testTurn?
So `CanFulfillIntentRequests` do not have `A: {responses} ` ??

Reply:
Yes, The prompts for `CanFulfillIntentRequest` is empty. 

The official doc for `CanFulfillIntentRequest` has some detailed explanation:
```
A CanFulfillIntentRequest request is made to a skill to query whether the skill can understand and fulfill the intent request with detected slots, before actually asking the skill to take action. The canFulfillIntent response that the skill provides should indicate whether or not the skill can fulfill the request, but the skill must not cause any observable interactions or make any changes in state outside of its scope except to return a value indicating whether the skill can fulfill the request. For example, the skill should not play sound, turn lights on or off, give the customer any feedback, or commit a transaction or a charge.
```

https://developer.amazon.com/en-US/docs/alexa/custom-skills/implement-canfulfillintentrequest-for-name-free-interaction.html

Reply:
Thanks for the explanation